test/mRides-server.Tests/UserControllerTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Moq;
+using mRides_server.Data;
+using mRides_server.Logic;
+using mRides_server.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Tests
+{
+    [TestFixture]
+    public class UserControllerTests    
+    {
+        UserCatalog catalog;
+        [SetUp]
+        public void Setup() {
+            int userId = 1;
+            var mokSet = initializeMokSet();
+
+            var mockContext = new Mock<ServerContext>();
+            mockContext.Setup(c => c.Users).Returns(mokSet.Object);
+            catalog = new UserCatalog(mockContext.Object);
+        }
+
+
+        [TearDown]
+        public void Tear() { }
+
+        [Test]
+        public void GetReview() 
+        {
+           
+            var feedbacks=catalog.getReviews(1);
+            foreach(var feedback in feedbacks)
+            {
+                if (feedback.givenAs == "rider")
+                {
+                    Assert.AreEqual(feedback.feedbackText,"Driver 2 to Rider 1");
+                }
+                else if (feedback.givenAs == "driver")
+                {
+                    Assert.AreEqual(feedback.feedbackText,"Rider 4 to Driver 1");
+                }
+            }
+            
+        }
+
+        public Mock<DbSet<User>> initializeMokSet()
+        {
+            var usersList = new List<User>();
+            usersList.Add(new User
```

Comment:
@nim442 This user object creation may be a bit too long and unclear since it even has nested object creation in it. Is there a better way to create a user instance to make it more readable?